x-pack/plugins/timelines/public/hooks/use_add_to_case.ts

```diff
@@ -109,25 +109,36 @@ export const useAddToCase = ({
     }
   }, [timelineById]);
   const {
-    application: { navigateToApp, getUrlForApp },
+    application: { navigateToApp, getUrlForApp, navigateToUrl },
     notifications: { toasts },
   } = useKibana<TimelinesStartServices>().services;
 
   const [isPopoverOpen, setIsPopoverOpen] = useState(false);
   const openPopover = useCallback(() => setIsPopoverOpen(true), []);
   const closePopover = useCallback(() => setIsPopoverOpen(false), []);
-  const isEventSupported = !isEmpty(ecsRowData.signal?.rule?.id);
+  const isAlert = useMemo(() => {
+    if (event !== undefined) {
+      const data = [...event.data];
+      return data.some(({ field }) => field === 'kibana.alert.uuid');
```

Comment:
Given all the changes going on, we should use the constant from `@kbn/rule-data-utils`